whois-nrtm4-client/src/main/java/net/ripe/db/nrtm4/client/config/NrtmDataSourceConfigurations.java

```diff
@@ -0,0 +1,30 @@
+package net.ripe.db.nrtm4.client.config;
+
+import com.mchange.v2.c3p0.ComboPooledDataSource;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.beans.PropertyVetoException;
+
+import static net.ripe.db.whois.common.DataSourceConfigurations.DRIVER_CLASS_NAME;
+import static net.ripe.db.whois.common.DataSourceConfigurations.createDataSource;
+
+@Configuration
+public class NrtmDataSourceConfigurations {
```

Comment:
Put nrtmv4 client condition here, datasource should only be configured  if it is enabled

Reply:
Rename to NrtmClientDataSourceConfigurations

Reply:
is not too redundant called all the objects client when they are already behing client module and package?

Reply:
I will put the condition, totally agree with that